vyxal/helpers.py

```diff
@@ -654,3 +654,16 @@ def wrapify(item: Any, count: int = None, ctx: Context = None) -> List[Any]:
             return [item]
         else:
             return item
+
+def pi_digits(x: int):
+    """Generate x digits of Pi. Spigot's formula."""
```

Comment:
Don't forget to wrap this in a lazylist - pushing raw generators doesn't work. Also, this needs a context parameter if it's going to be used as an element function.